### PR TITLE
feat: API changes needed for "real" draft imports

### PR DESCRIPTION
**Pull request by jennifer-richards**
This is the datatracker side of https://github.com/ietf-tools/rpc/pull/178

### CODE REVIEW

File: ietf/api/views_rpc.py [hunk 1]
```diff
@@ -83,16 +97,17 @@ def rpc_draft(request, doc_id):
...
                 {
                     "id": p.pk,
                     "plain_name": p.person.plain_name(),
                 }
                 for p in d.documentauthor_set.all()
             ],
-        }
-    )
-
+        "shepherd": d.shepherd.formatted_ascii_email() if d.shepherd else "",
```
Comment (rjsparks) [inline ietf/api/views_rpc.py:108]: See my comment on the companion RPC PR - I'd rather not leak this information into the RPC app unless we really need to. The field will go through a big model change soon.

**Review by rjsparks — APPROVED** [review]
Approved with one reservation ☝️

File: ietf/api/views_rpc.py (hunk 1 above)
Comment (ajeanmahoney) [reply ietf/api/views_rpc.py:108]: The Document Shepherd email address is included in the CC of the AUTH48 messages

Comment (rjsparks) [reply ietf/api/views_rpc.py:108]: We could fetch that directly from the datatracker when generating the message.

Comment (jennifer-richards) [reply ietf/api/views_rpc.py]: Leaving the shepherd email address in the API so it can be shown, but no longer storing it on the RPC side at all.

**Pull request merged**